terraform-modules/k8s/k8s-node-pool.tf

```diff
@@ -29,7 +29,8 @@ resource "google_container_node_pool" "node-pool" {
 
     # Protect node metadata
     workload_metadata_config {
-      node_metadata = var.workload_metadata_config_node_metadata
+      # Workload Identity only works when using the metadata server.
+      node_metadata = var.enable_workload_identity ? "GKE_METADATA_SERVER" : "SECURE"
```

Comment:
Does anyone know if there's a hard AppSec requirement to use `SECURE` here? Vs. a requirement to not use `EXPOSED`? This feature only works when using the GKE metadata server.

Reply:
Security is a case by case basis, but seems fine to me.

Reply:
Agree--case-by-case, the default is secure and that's nice.